models/product/src/product/index.ts

```diff
@@ -21,9 +20,11 @@ export const ProductGraphql = {
   presets: modelPresets.graphqlPresets,
 };
 
-export * as Product from '.';
```

Comment:
This way of exporting was quite misleading so I believe it's better to explicitly export an object so we can see what we are exporting.